docs/general/data-structures.md

```diff
@@ -85,6 +85,20 @@ options: {
 }
 ```
 
+When using keys of the form `foo.bar`, you can allow for escaping dot symbol and add a double slash before the dot.
+
+```javascript
+type: 'doughnut',
```

Comment:
Change this to line or bar since the doughnut chart does not use xAxisKey and yAxisKey